TODO.txt

```diff
@@ -0,0 +1,68 @@
+TODO C++ Papas Code. Aug  2016
+
+
+FCCSW Guidelines & Code Formatting
+Clear out debugging stuff and commented out sections 
+Ask Joshka: See if there are new tools for automated checking
+ASk Joschka: How best to integrate DOxygen outputs
+
+Commenting
+Still more to do but it has improved.
+
+ROOT: 
+ask Pere re Makefiles
+	Ongoing problem with ROOT and Xcode not working using FCCSW cmakelists
+	Figure out how to get DYLD_LIBRARY_PATH properly sorted for Mac - need advise.
```

Comment:
could you tell me more about that? you can see examples about how to manage the DYLD lib path in other FCCSW packages such as fcc-edm